### DISCUSSION TITLE
Motion mask time

### DISCUSSION THREAD

**Discussion by Hubsi4** [gopro/labs#585] [Q&A]
"Motion Mask:    seconds, to avoid falsely detecting constantly moving objects like a ceiling fan, or winds blowing leaves. Anything moving during this motion mask time will be ignored as motion triggers."

Please explain in a little more detail, i do not understand fully. When I set to 1 sec, does it trigger only objects moving longer than one second? Leafes in the wind normaly move constantly, so longer than 1 sec.

**Comment by dnewman-gpsw** [comment]
Any movement detected with the mask time, will be excluded.  Leaves are that hardest to mask, as the can move further under more wind, so it better to mask potential areas of movement using your hand in front of the lens, watching the screen to paint out risk areas.
